### PR TITLE
Bump CI image versions for 1.13 release.

### PR DESCRIPTION
**Pull request by cjwagner**
This PR will address steps 2-7 from https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/test-infra#create-cipresubmit-jobs-for-the-new-release
/sig release
/cc @krzyzacy @amwat @imkin @rahulmahale

### CODE REVIEW

**Comment by krzyzacy** [comment]
also drop 1.9?

**Comment by cjwagner** [comment]
If this looks good to @krzyzacy I will now run
```shell
experiment/bump_e2e_image.sh
make -C images/kubekins-test push
```

**Comment by cjwagner** [comment]
> also drop 1.9?

Done.

**Comment by krzyzacy** [comment]
lgtm, it doesn't cover step 4 yet, though

**Comment by cjwagner** [comment]
> lgtm, it doesn't cover step 4 yet, though

Yeah, I think I need to create the new images (commands above) before I can do 4, but I do intend to include those changes in this PR.

**Comment by krzyzacy** [comment]
sgtm, do you intend to cover image tag changes for release branch jobs in this PR or a separate PR?

**Comment by cjwagner** [comment]
I'll do that in this PR.

**Comment by cjwagner** [comment]
@krzyzacy It looks like the image bump script has a bug. 
It appears to have updated image tags for every use of kubekins-e2e, not just the versions that are currently specified. In other words the script created a bunch of tags like `v20181105-2c60bd0e3-1.9` and `v20181105-2c60bd0e3-1.8` even though those images don't exist.

**Comment by krzyzacy** [comment]
@cjwagner looks like those are jobs belong to 1.9 branch, step 5 will cover those. (the bump script won't change image branches)

**Comment by cjwagner** [comment]
> @cjwagner looks like those are jobs belong to 1.9 branch, step 5 will cover those. (the bump script won't change image branches)

Only a few of the image 1.9 image tags that were bumped were associated with a job marked with `#manual-release-bump-required`.

**Comment by cjwagner** [comment]
This should be ready for review now.
/hold

File: config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-upgrade.yaml [hunk 1]
```diff
@@ -91,7 +92,38 @@ periodics:
       - --skew
       - --test_args=--ginkgo.focus=\[Conformance\]|\[Feature:BootstrapTokens\]|\[Feature:NodeAuthorizer\] --minStartupPods=8
       - --timeout=300m
-      - --upgrade_args=--ginkgo.focus=\[Feature:MasterUpgrade\] --ginkgo.skip=statefulset-upgrade|hpa-upgrade|service-upgrade --upgrade-target=release/latest-1.9
+      - --upgrade_args=--ginkgo.focus=\[Feature:MasterUpgrade\] --ginkgo.skip=statefulset-upgrade|hpa-upgrade|service-upgrade --upgrade-target=release/latest-1.12
+
+- name: ci-kubernetes-e2e-kubeadm-gce-upgrade-1-12-1-13
```
Comment (krzyzacy) [inline config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-upgrade.yaml]: cc @neolit123 to double check if this looks correct

File: images/kubekins-test/Dockerfile-1.9 [hunk 2]
```diff
@@ -1,54 +0,0 @@
-# Copyright 2017 The Kubernetes Authors.
```
Comment (krzyzacy) [inline images/kubekins-test/Dockerfile-1.9]: (I don't see a 1.13 Dockerfile?)

**Comment by krzyzacy** [comment]
thanks @cjwagner! maybe also update https://github.com/kubernetes/test-infra#release-branch-jobs--image-validation-jobs? otherwise looks good besides the kubekins-test image for 1.13.

File: images/kubekins-test/Dockerfile-1.9 (hunk 2 above)
Comment (cjwagner) [reply images/kubekins-test/Dockerfile-1.9]: It already existed before this PR: https://github.com/kubernetes/test-infra/blob/master/images/kubekins-test/Dockerfile-1.13

**Comment by cjwagner** [comment]
> maybe also update https://github.com/kubernetes/test-infra#release-branch-jobs--image-validation-jobs?

Done.

**Comment by krzyzacy** [comment]
cc @AishSundar @jberkus

**Comment by AishSundar** [comment]
Thanks @cjwagner 

We are delaying Beta for 1.13. This PR will go in post the branch cut right?

/lgtm
/hold

**Comment by cjwagner** [comment]
> We are delaying Beta for 1.13. This PR will go in post the branch cut right?

Correct. This PR is dependent on the `release-1.13` branch existing in `k/k`.

File: config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-upgrade.yaml (hunk 1 above)
Comment (neolit123) [reply config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-upgrade.yaml]: the kubeadm changes LGTM.

**Comment by cjwagner** [comment]
Rebased, please restamp!
/hold cancel

**Pull request merged**